os/network-config-with-networkd.md

```diff
@@ -70,8 +70,6 @@ By default, even if you've already set a static IP address and you have a workin
 
 coreos:
   units:
-    - name: systemd-networkd.service
-      command: stop
```

Comment:
networkd needs to be stopped before we take down the interfaces. Otherwise, networkd will bring them right back up. This whole this is a shitty pile of hacks and we should really focus on Ignition instead.